app/build.gradle

```diff
@@ -20,16 +20,18 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:26.1.0'
+    implementation 'com.android.support:appcompat-v7:27.0.2'
```

Comment:
Same here, just make sure Ivan's PR is the one having 27.0.2, if it is we will be fine

Reply:
Ivan's code have 26.1.0 so I undo it to 26.1.0

Reply:
be sure it is still working after that change, if it isn't make ivan test with 27.0.2... we want ivan code to work